utils/KoalaVoteUtils.py

```diff
@@ -0,0 +1,176 @@
+class TwoWay(dict):
```

Comment:
Ideally, all cog related items should be within the Voting.py file to continue with the modularity approach to Koala. If there is any reason against doing this, let me know